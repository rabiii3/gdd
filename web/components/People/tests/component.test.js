import React from 'react';
import { shallow } from 'enzyme';
import { Component } from '..';

describe('web | components | Peoples | index ', () => {
  describe('render', () => {
    it('should match snapshot', () => {
      const props = {
        people: [],
      };
      const wrapper = shallow(<Component {...props} />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
