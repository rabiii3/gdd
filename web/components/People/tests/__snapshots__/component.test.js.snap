// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`web | components | Peoples | index  render should match snapshot 1`] = `
<WithStyles(Grid)
  container={true}
  justify="center"
  wrap="wrap"
/>
`;
